### PR TITLE
feat: Add notebook for OpenAI/gpt-3

### PR DESCRIPTION
**Pull request by niehaus59**
# Summary

Add notebook for OpenAI / GPT-3

# Tests

N/A

# Dependency changes

N/A

AB#1877573

### CODE REVIEW

File: notebooks/features/cognitive_services/CognitiveServices - OpenAI.ipynb [hunk 1]
```diff
@@ -0,0 +1,846 @@
...
+   ],
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
```
Comment (niehaus59) [inline notebooks/features/cognitive_services/CognitiveServices - OpenAI.ipynb:23]: Should all the metadata sections be empty? I noticed they are empty in the other notebooks.

**Review by mhamilton723 — DISMISSED** [review]

**Review by mhamilton723 — APPROVED** [review]

**Pull request merged**